### ISSUE TITLE
Windows version not working?

### ISSUE THREAD

**Issue by karsten-wagner** [apigee/apigee-remote-service-cli#115]
**Describe the issue**
Running the windows version of the CLI command (v.1.2.0) on my PC does not provision the remote-service proxy and ends with HTTP 400 response from the Apigee Management API (intercepted via local proxy server, not shown in the verbose output):
```JSON
{ "code" : "messaging.config.beans.InvalidBundle", "message" : "Bundle is invalid. Empty bundle", "contexts" : [ ] }
```

**Steps to Reproduce**
```bash
$ ./apigee-remote-service-cli.exe provision --legacy --token "${TOKEN}" --organization "${ORG}" --environment "${ENV}" --verbose > config.yaml

checking if proxy remote-service deployment exists...
GET /v1/organizations/<org>/environments/<env>/apis/remote-service/deployments HTTP/1.1
Host: api.enterprise.apigee.com
User-Agent: go-apigee-edge/0.1.0
Accept: application/json
Authorization: ***
Accept-Encoding: gzip



HTTP/1.1 404 Not Found
Content-Length: 176
Connection: keep-alive
Content-Type: application/json
Date: Fri, 20 Nov 2020 16:22:59 GMT
Server: Apigee LB

{
  "code" : "messaging.config.beans.ApplicationDoesNotExist",
  "message" : "APIProxy named remote-service does not exist in organization <org>",
  "contexts" : [ ]
}

checking proxy remote-service status...
GET /v1/organizations/<org>/apis/remote-service HTTP/1.1
Host: api.enterprise.apigee.com
User-Agent: go-apigee-edge/0.1.0
Accept: application/json
Authorization: ***
Accept-Encoding: gzip



HTTP/1.1 404 Not Found
Content-Length: 176
Connection: keep-alive
Content-Type: application/json
Date: Fri, 20 Nov 2020 16:22:59 GMT
Server: Apigee LB

{
  "code" : "messaging.config.beans.ApplicationDoesNotExist",
  "message" : "APIProxy named remote-service does not exist in organization <org>",
  "contexts" : [ ]
}

creating new proxy remote-service revision: 1...
Error: deploying proxy remote-service: importing proxy remote-service: POST https://api.enterprise.apigee.com/v1/organizations/<org>/apis?action=import&name=remote-service: 400 {0  }
Usage:
  apigee-remote-service-cli provision [flags]

Flags:
      --analytics-sa string    path to the service account json file (for GCP-managed analytics only)
  -c, --config string          path to Apigee Remote Service config file
  -e, --environment string     Apigee environment name
  -f, --force-proxy-install    force new proxy install (upgrades proxy)
  -h, --help                   help for provision
      --insecure               allow insecure server connections when using SSL
      --legacy                 Apigee SaaS (sets management and runtime URL)
  -m, --management string      Apigee management base URL (default "https://apigee.googleapis.com")
      --mfa string             Apigee multi-factor authorization token (legacy only)
  -n, --namespace string       emit configuration in the specified namespace (default "apigee")
      --opdk                   Apigee opdk
  -o, --organization string    Apigee organization name
  -p, --password string        Apigee password (legacy or OPDK only)
      --rotate int             if n > 0, generate new private key and keep n public keys (hybrid only)
  -r, --runtime string         Apigee runtime base URL (required for hybrid or opdk)
  -t, --token string           Apigee OAuth or SAML token (overrides any other given credentials)
  -u, --username string        Apigee username (legacy or OPDK only)
  -v, --verbose                verbose output
      --virtual-hosts string   override proxy virtualHosts (default "default,secure")
```

**Platform (include version)**
OS: Windows 10, Version 1909, OS Build 18363.1082
Envoy: N/A
Kubernetes: N/A
Istio: N/A

**Additional context**
I have tried exactly the same call on WSL Ubuntu 20.04, which is working like a charm. So I can exclude any issues with the proxy server, which is the same in all cases.
The command itself is executed via bash, compiled for Windows (MSYS2), but I have also tried this natively on Windows cmd with the same result.
I have also downloaded the repository at tag v1.2.0, compiled the go source and ended up in the same error. I have also tried to debug the go code, but did not understand where exactly the request fails (mea culpa).
I have also tried versions 1.0.0, 1.1.0 and 1.3.0-rc1, all not successful (looks like the "token" parameter is only supported as of 1.2.0, however all failed...

**Comment by rockspore** [comment]
@karsten-wagner Thanks a lot for pointing this out with the informative details! It is indeed a bug - the back slashes used for zip file names were not correctly replaced with forward slashes.

The fix, which I am going to submit for PR, is simple. Hopefully, we will resolve this issue in 1.3.0.

Thanks again!

**Comment by theganyo** [comment]
We are on target to we release 1.3 early next week. This fix will be included. Thank you!

**Comment by karsten-wagner** [comment]
@theganyo, @rockspore, thanks for the quick fix.

I have verified the latest version on master compiles well and is able to provision the service with the command I shared in the issue description. I have also cherry-picked the fix and applied it on top of tag v1.2.0, which also compiles well and then successfully deploys the remote service. You might eventually want to provide a patch version v1.2.1?

Thanks a lot for the quick turnover - you guys rock!
